### ISSUE TITLE
Import MEGA Files

### ISSUE THREAD

**Issue by AABoyles** [CDCgov/MicrobeTrace#47]
**Background**
MEGA is one of the most ubiquitous biological sequence viewing, editing and alignment tools. Accepting MEGA output files would broaden MicrobeTrace's user base to include academics and international scholars. Additionally, by accepting MEGA output, users can leverage a more robust multiple sequence alignment to include more complex sequence data. 

**Open Dask Description**
MicrobeTrace ingests [FASTA files](https://github.com/CDCgov/MicrobeTrace/wiki/FASTA-Files) for sequence data. A similar file format is the [MEGA Sequential data format](https://www.megasoftware.net/web_help_7/rh_converting_fasta_format.htm). It would be nice for MicrobeTrace to be able to import MEGA Sequential data files.

To accomplish this:
1. Take a look at the [`app.parseFASTA` function](https://github.com/CDCgov/MicrobeTrace/blob/master/scripts/common.js#L234) (located in the [`scripts/common.js` file](https://github.com/CDCgov/MicrobeTrace/blob/master/scripts/common.js). Copy it to `app.parseMEGA` and modify the copy to look for `#` instead of `>`. Also, devise a check for the MEGA Header (FASTA files don't have headers).
2. In `components/files.html`, once the user [clicks "submit"](https://github.com/CDCgov/MicrobeTrace/blob/master/components/files.html#L445), add a check for whether a file is a MEGA file or not. If it is, do exactly the same thing you would have with a FASTA file, only use `app.parseMEGA` instead of `app.parseFASTA`.

**Comment by GrooveCS** [comment]
Is this still an Open Issue?

**Comment by AABoyles** [comment]
Yes it is! PR #184 resolved the first half of the issue (creating the MEGA parser), but didn't address the second (integrating it into the application logic).

**Comment by GrooveCS** [comment]
Are you open if we can  provide support?

**Comment by ells** [comment]
@GrooveCS Thank you for asking, we welcome open source contributions on all of our issues! Feel free to interact with us here or via email at microbetrace@cdc.gov.

**Comment by AABoyles** [comment]
@GrooveCS Agreed, please do! You'll make our day ;)
